usr/share/rear/restore/BAREOS/default/400_restore_backup.sh

```diff
@@ -1,178 +1,114 @@
-### Restore from bareos
-###
-
-### Create example bootstrap file
-cat <<EOF >$VAR_DIR/bootstrap.txt
-### Example Bareos bootstrap file
-###
-
-### Only the exact Volume name is required, other keywords are optional.
-### The use of FileIndex and Count keywords speeds up the selection enormously.
-
-### The Volume name to use
-Volume=PLEASE-EDIT-BOOTSTRAP
-
-### A (list of) Client name(s) to be matched on the current Volume
-#Client=$(hostname -s)-fd
-
-### A (list or range of) JobId(s) to be selected from the current Volume
-#JobId=18
-
-### A (list of) Job name(s) to be matched on the current Volume
-#Job=Bkp_Daily.2011-06-16
-
-### A (list or range of) Volume session id(s) to be matched from the current Volume
-#VolSessionId=1
-
-### The Volume session time to be matched from the current Volume
-#VolSessionTime=108927638
-
-### A (list or range of) FileIndex(es) to be selected from the current Volume
-#FileIndex=1-157
-
-### The total number of files that will be restored for this Volume.
-#Count=157
+#
+# Restore from Bareos using bconsole (optional)
+#
+
+# get last restore jobid before starting the restore
+local last_restore_jobid_old
+last_restore_jobid_old=$(get_last_restore_jobid)
+local restore_jobid
+
+echo "status client=$BAREOS_CLIENT" | bconsole > "$TMP_DIR/bareos_client_status_before_restore.txt"
+LogPrint "$(cat "$TMP_DIR/bareos_client_status_before_restore.txt")"
+
+LogPrint "target_fs_used_disk_space=$(total_target_fs_used_disk_space)"
```

Comment:
Isn't this always `0`because ReaR freshly formatted everything?

Reply:
On my test machine, it starts with 42k (meta-data). I think, it can be helpful to see that the disk did not start with 0.